content/release/1.3.0.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Release 1.3.0 available
+date: 2021-11-30
+linked: true
+aliases:
+    - /release/1.3.0
+---
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+![glacier-haiku](releases/1.3.0.jpg)
```

Comment:
Not sure how useful the alt text "glacier-haiku" is, but it should refer to Grand Canyon.